### PR TITLE
Command becomes struct.

### PR DESCRIPTION
**Pull request by dairyo**
Related PR: #296

### CODE REVIEW

File: ThingIFSDK/ThingIFSDK/Command.swift [hunk 1]
```diff
@@ -5,70 +5,56 @@
...
+    public let commandID: String
     /** ID of the Command Target. */
-    open let targetID: TypedID
+    public let targetID: TypedID
     /** ID of the issuer of the Command. */
-    open let issuerID: TypedID
+    public let issuerID: TypedID
     /** Actions to be executed. */
-    open let actions: [AliasAction]
+    public let actions: [AliasAction]
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFSDK/Command.swift]: can you change name to `aliasActions`

File: ThingIFSDK/ThingIFSDK/Command.swift [hunk 2]
```diff
@@ -5,70 +5,56 @@
...
     /** Results of the action. */
-    open let actionResults: [AliasActionResult]
+    public let actionResults: [AliasActionResult]
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFSDK/Command.swift]: can you change name to `aliasActionResults`?

File: ThingIFSDK/ThingIFSDK/Command.swift [hunk 3]
```diff
@@ -5,70 +5,56 @@
...
+
+    /** Initialize `Command`.
 
-    internal init(_ commandID: String,
+     Developers rarely use this initializer. If you want to recreate
+     same instance from stored data or transmitted data, you can use
+     this method.
+
+     - Parameter commandID: ID of the Command.
+     - Parameter targetID: ID of the Command Target.
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFSDK/Command.swift]: add parameter `issuerID`

File: ThingIFSDK/ThingIFSDK/Command.swift [hunk 4]
```diff
@@ -100,7 +86,7 @@ open class Command: NSObject, NSCoding {
      - Parameter alias: Alias to get action.
      - Returns Array of `AliasAction`.
      */
-    open func getAction(_ alias: String) -> [AliasAction] {
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFSDK/Command.swift:103]: method name should be `getAliasAction`

**Review by yongpingchen — CHANGES_REQUESTED** [review]

File: ThingIFSDK/ThingIFSDK/Command.swift (hunk 4 above)
Comment (dairyo) [reply ThingIFSDK/ThingIFSDK/Command.swift:103]: I think this method name `getAction`.

This is a method corresponding to following java method:
```java
    public <T extends Action> List<AliasAction<T>> getAction(
            @NonNull String alias,
            @NonNull Class<T> clsOfT)
```

What do you think?

Comment (yongpingchen) [reply ThingIFSDK/ThingIFSDK/Command.swift:103]: I think they are different method. The return value is different

Comment (dairyo) [reply ThingIFSDK/ThingIFSDK/Command.swift:103]: @yongpingchen 
I got it. This method is wrong. I should implement a method like following

```swift
public func getAction(_ alias: String, actionName: String) -> [AliasAction]
```

What do you think?

Comment (yongpingchen) [reply ThingIFSDK/ThingIFSDK/Command.swift:103]: @shinsuke-nara ah. I found the `getAction` in Android is wrong. Should be 
```java
    public <T extends Action> List<AliasAction<T>> getAliasAction(
            @NonNull String alias)
```

**Pull request merged**